source/spirv_target_env.h

```diff
@@ -27,6 +27,9 @@ bool spvIsVulkanEnv(spv_target_env env);
 // Returns true if |env| is an OPENCL environment, false otherwise.
 bool spvIsOpenCLEnv(spv_target_env env);
 
+// Returns true if |env| is an WEBGPU environment, false otherwise.
```

Comment:
WebGPU ? I'm not sure the right capitalization here, what does the spec say?

Reply:
The other functions of this type in this header are documented using full caps. I think the intent is to indicate that this applies to environment enums that have the string WEBGPU (or VULKAN or OPENCL) in them.